apps/web-connect/src/features/checkout/components/ActionSection.tsx

```diff
@@ -44,12 +45,12 @@ export function ActionSection(): JSX.Element {
      * @returns {string} The button text
      */
     const getTokenButtonText = useCallback(() => {
-        if (mintWithEth.isLoading || mintWithXai.isLoading || approve.isLoading) return "WAITING FOR CONFIRMATION";
-        if (chain?.id !== 42161) return "Please Switch to Arbitrum One";
+        if (mintWithEth.isLoading || mintWithXai.isLoading || approve.isLoading) return "WAITING FOR CONFIRMATION..";
+        // if (chain?.id !== 42161) return "Please Switch to Arbitrum One";
```

Comment:
Same as the comment bellow.

We only would remove these checks on a sepolia testnet build.

Reply:
I believe these are all cleaned up now. Thank you for the guidance!